bin/logstash.lib.sh

```diff
@@ -1,5 +1,29 @@
 unset CDPATH
-LOGSTASH_HOME=$(cd `dirname $0`/..; pwd)
+# This unwieldy bit of scripting is to try to catch instances where Logstash
```

Comment:
Because 

```
cd `dirname $0`/..
```

was in both files, and after I fixed the first, it still wouldn't work until the second _also_ had it.